src/main/kotlin/no/nav/modiapersonoversikt/Configuration.kt

```diff
@@ -8,8 +8,8 @@ private val defaultProperties = ConfigurationMap(
                 "NAIS_CLUSTER_NAME" to "local",
                 "ISSO_JWKS_URL" to "https://isso-q.adeo.no/isso/oauth2/connect/jwk_uri",
                 "ISSO_ISSUER" to "https://isso-q.adeo.no:443/isso/oauth2",
-                "DATABASE_JDBC_URL" to "jdbc:h2:mem:testdb;MODE=PostgreSQL;DB_CLOSE_DELAY=-1",
-//                "DATABASE_JDBC_URL" to "jdbc:h2:tcp://localhost:8090/./testdb",
+//                "DATABASE_JDBC_URL" to "jdbc:h2:mem:testdb;MODE=PostgreSQL;DB_CLOSE_DELAY=-1",
+                "DATABASE_JDBC_URL" to "jdbc:h2:tcp://localhost:8090/./testdb",
```

Comment:
Denne skal vel bort?

Reply:
Påvirker bare hva slags database man har ved lokal kjøring. Var tidligere en in-memory databasen, nå er det databasen som kommer opp ved å kjøre h2local.kt. Verdien blir uansett overskredet når man kjører på naid

Reply:
Ahaaaa!